_data/sites.json

```diff
@@ -22,6 +22,18 @@
         ]
     },
 
+    {
+        "name": "123RF",
+        "url": "https://www.123rf.com/support/support-answer.php?id=8817",
+        "difficulty": "hard",
+        "notes": "Only way to delete your account is sending them an e-mail.",
+        "email": "submission@123rf.com",
+        "email_body": "I want my account to be deleted. My full name is XXXXXX and my user name is XXXXXX I want my account to be deleted because of XXXXXX. I request and authorize you to remove my profile from 123rf.com and delete all my content with you. I wish to terminate the 123RF Contributor Agreement and agree to the Terms and Conditions as set in the Contributor Agreement for the termination process.",
```

Comment:
```suggestion
        "email_body": "I want my account to be deleted. My full name is XXXXXX and my user name is XXXXXX, I want my account to be deleted because of XXXXXX. I request and authorize you to remove my profile from 123rf.com and delete all my content with you. I wish to terminate the 123RF Contributor Agreement and agree to the Terms and Conditions as set in the Contributor Agreement for the termination process.",
```